bin/zopen-importenvs

```diff
@@ -1,18 +1,87 @@
-#!/bin/sh
-#
+#!/bin/bash
 # Used for development to source all the boot .envs
-# . importenvs
+#. ./zopen-importenvs <path to buildenv to fetch dependency>
 
-if [ ! -d "${HOME}/zopen/boot/" ]; then
-  echo "boot directory does not exist.  Existing"
-  exit 4;
-fi
 
-for deps in $(find "${HOME}/zopen/boot/" -name ".env")
-do
-  echo "Found ${deps}"
-  depdir=$(dirname "${deps}")
-  cd $depdir
-  . ./.env
-  cd -
-done
+#below code fetches the ~/code/utils/bin path where the common.inc file is under its lib dir
```

Comment:
Suggest adding a space between # and 'below' for comment consistency

Reply:
I have modified this!